### PR TITLE
Add quick installation instruction to README.

### PR DESCRIPTION
**Pull request by laurentlb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,15 @@ For ease of use, the Python version of Bazelisk is written to work with Python 2
 
 The Go version can be compiled to run natively on Linux, macOS and Windows.
 
+To install the Go version, you first need to set the [`GOPATH` environment
```
Comment (philwo) [inline README.md]: Thanks!

I think recent versions of Go no longer require setting this (I don't :)).

It defaults to reasonable places on all platforms:
> $HOME/go on Unix, $home/go on Plan 9, and %USERPROFILE%\go (usually C:\Users\YourName\go) on Windows. (https://golang.org/doc/code.html#GOPATH)

Might be nice to mention that setting PATH to include `$GOPATH/bin` makes it easier to use:

> For convenience, add the workspace's bin subdirectory to your PATH:
> `export PATH=$PATH:$(go env GOPATH)/bin`

Comment (laurentlb) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -38,6 +38,21 @@ For ease of use, the Python version of Bazelisk is written to work with Python 2
...
 
+To install the Go version, type:
+
+```shell
+go get github.com/philwo/bazelisk
+```
+
+To add it to your PATH:
+
+```python
```
Comment (philwo) [inline README.md]: s/python/shell/ 😊

Otherwise LGTM, thanks!

Comment (philwo) [reply README.md]: I'll just merge and fix this quickly...

**Pull request merged**